example/src/main/java/org/blockstack/android/MainActivity.kt

```diff
@@ -283,13 +270,8 @@ class MainActivity : AppCompatActivity() {
         }
 
         if (intent?.action == Intent.ACTION_VIEW) {
-            lifecycleScope.launch(Dispatchers.Main) {
-                val response = BlockstackConnect.handleAuthResponse(intent)
-                if (response.value != null) {
-                    runOnUiThread {
-                        onSignIn(response.value!!)
-                    }
-                }
```

Comment:
Isn't this code needed?

Reply:
no longer, we were using connect to both Connect and Browser, that could result in a few issues with Browser. The generic solution is "better" :)